apps/landing/src/content/blog/removemos-o-crm.md

```
@@ -0,0 +1,133 @@
+2. **Benchmark público reduz viés interno.**
+3. **Tom importa.** Remoção precisa ser comunicada com humildade, sem tom de superioridade e com respeito aos usuários atuais.
+
+Na prática, clientes que usavam o módulo estão em acompanhamento específico, e o ponto de continuidade está registrado no `MON-1106` para não virar apenas “apagamos e ignoramos”.
+
+## CTA
+
+Não foi uma decisão para parecer ousado. Foi para ficar mais consistente.
+
+Se você curte produto que assume trade-offs com transparência, esse é o tipo de decisão que nos permite ser melhores no que já é o nosso: entregar finanças mais claras, rápidas e úteis.
+
+Se quiser acompanhar o que está por vir, o link canônico está aqui: **/blog/removemos-o-crm**.
```

markdownlint-cli2 (0.22.1)

[warning] Files should end with a single newline character

(MD047, single-trailing-newline)